structurizr-core/test/unit/com/structurizr/io/plantuml/PlantUMLWriterTests.java

```diff
@@ -11,6 +11,8 @@
 import static org.junit.Assert.assertEquals;
 
 public class PlantUMLWriterTests {
+    private static final String DATASTORE = "DataStore";
+    private static final String SOME = "Some";
```

Comment:
Hello, this is my first time reviewing an open source project. Is it fine to do so just because I want to? If yes, I'll write some thoughts.

These contants are being used as Tags deep below, so calling them `DATABASE_TAG` or similar would help.

And what about `SOME` tag? Is it used for something? Is it a shortcut? If it's not being used, couldn't it be removed?

Reply:
Happens to be my first pull request for /this/ project too so not sure what the project-specific etiquette  is - but I'm more than happy to take the feedback :)

Renaming the constants for clarity makes sense - hope to address this evening. The SOME tag was used to mark the existing `SomeController` and `SomeRepository` and in `test_writeComponentsView()` is used to highlight those components yellow.

Reply:
Well, I didn't realize this was a test. I see now. I would name the it `SOME_TEST_TAG` or just `SOME_TAG`.